package.json

```diff
@@ -72,7 +71,7 @@
     "regenerator-runtime": "^0.13.7",
     "rudder-component-cookie": "0.0.1",
     "semver": "^7.1.3",
-    "set-value": "^4.1.0",
+    "set-value": "3.0.2",
```

Comment:
Is this change still required?

Reply:
I didn't see the changes in other places for not using the package. As we discussed, it is not a recommended fix.

Reply:
done